src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseFilesService.cs

```diff
@@ -817,31 +854,19 @@ private string AdminReleasePathWithFileReference(ReleaseFileReference fileRefere
                 fileReference.ReleaseFileType == ReleaseFileTypes.Chart ? fileReference.Id.ToString() : fileReference.Filename);
         }
 
-        private async Task<string> GetFilenameAssociatedToType(Guid releaseId, string filename, ReleaseFileTypes type, ReleaseFileTypes associatedType)
```

Comment:
Nice, I was wondering what all of this stuff was doing when I was refactoring all the blob storage stuff...

Reply:
👍